tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Basic Options */
     "target": "es2016" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
```

Comment:
I wonder if we should keep `target` in sync with `lib` here

Reply:
Thanks. Updated.